### DISCUSSION TITLE
Always show scrollbar?

### DISCUSSION THREAD

**Discussion by rieje** [piroor/treestyletab#3124] [Q&A] [answered]
Is there a way to possible show scrollbar? Currently, it auto-hides after the cursor leaves the sidebar for a second. I typically many tabs and always seeing the scrollbar position is helpful for context for general browsing.

**Comment by piroor** [comment, marked as answer]
Do you use Linux or Windows 11? Firefox 100 and later on those platforms had the change:

https://www.mozilla.org/en-US/firefox/100.0/releasenotes/

> Scrollbars on Linux and Windows 11 won't take space by default. On Linux, users can change this in Settings. On Windows, Firefox follows the system setting (System Settings > Accessibility > Visual Effects > Always show scrollbars).

Currently I'm using Firefox + TST on Windows 10 and I've not found any solution yet...

**Reply by piroor** [reply]
https://www.reddit.com/r/firefox/comments/ujo1xy/how_to_increase_firefox_100_scrollbar_width/i7wgedo/?utm_source=reddit&utm_medium=web2x&context=3
I've found the way. You need to change a setting of Firefox itself or the one of Windows 11:

> If you're on Linux you can change it in Firefox's settings (Preferences > General > Navigation > Always show scrollbars).
> 
> On Windows it follows a system setting (Settings > Accessibility > Visual Effects > Always show scrollbars).

**Reply by piroor** [reply]
I've introduced a change 8eb3c0814 to apply padding to tabs. When the sidebar become scrollable, twisty in tabs are shifted to avoid covered by the scrollbar.

**Reply by rieje** [reply]
Oh thanks, that works. I guess Firefox scrollbar for webpage and for scrollbar for tabs on TST are treated the same. I would prefer to keep scrollbar for webpage to only "smartly show" but it's not a big deal. I'll close this.
